src/app/components/Copilot/Copilot.tsx

```diff
@@ -76,9 +76,11 @@ export function createNewConversation() {
 export async function askQuestion({
                                     query,
                                     relevant,
+                                    // model,
                                   }: {
   query: string;
   relevant: string;
+  // model: string;
```

Comment:
this is correct here to define this for the askQuestion() - then you can pass in the model ID where you have the `model: selectedModel`. You will just have to only use the cloud models here (for now) since we are not handling the download and usage of any local model. 

Using the local models requires the modelProgressDownloadController file that you can see an example of [here ](https://github.com/pieces-app/pieces-copilot-vanilla-typescript-example/blob/main/src/CopilotStreamController.ts) to monitor their download status so they can actually be passed into your question logic. 

For now i would just stick to the cloud models: 
* OpenAI models (4 models)
* Palm2 models (2 models)
* gemini models (1 model)

Reply:
@Arindam200